src/lib/Tab/LocationView/AuthorsTab.php

```diff
@@ -92,7 +85,7 @@ public function getTemplateParameters(array $contextParameters = []): array
 
     public function evaluate(array $parameters): bool
     {
-        return IsFocusModeEnabled::fromUserSettings($this->userSettingService)->isSatisfiedBy(FocusMode::FOCUS_MODE_ON);
+        return true;
```

Comment:
IMHO this change is invalid and it's probably specification issue. Authors tab is excluded as the same data (and more) is duplicated in Versions Tab

Reply:
@radzyu 
Could you look at it, at verify/specify issue?

Reply:
@ViniTou & @adamwojs I verified this issue with @SylvainGuittard and Ida. And we agreed, that 'Authors' tab should be visible in both: enabled and disabled Focus mode. This tab shouldn't disappear, when you turn off Focus mode. 
When it comes to the data displayed, Authors tab was created (by splitting old Details tab), to improve UX and to make information about contributor and creator (and dates) more visible and accessible. If you have multiple published versions it's harder to recognize them. Additionally, we store and display a limited number of versions as Sylvain mentioned.

Design: https://www.figma.com/file/e2G3D6kwy6khoDXlqIxxbq/Focus-mode-%26-site-context?type=design&node-id=2598%3A86272&mode=design&t=byXUOKQglTbDnDk4-1

Reply:
@radzyu please make sure this is covered in requirements :wink:

Reply:
@adamwojs it was in specification and design: https://ibexa.atlassian.net/wiki/spaces/Product/pages/168132705/Administration+UI+UX+improvements#%3Ablue_star%3A--Functional-Requirements*